firefox-ios/Client/Frontend/Onboarding/Models/OnboardingCardInfoModelProtocolExtension.swift

```diff
@@ -0,0 +1,36 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import OnboardingKit
+
+extension OnboardingKit.OnboardingCardInfoModelProtocol
```

Comment:
Note: This is an extension of the generic OnboardingCardInfoModelProtocol that resides in the OnboardingKit package. Since we have OnboardingMultipleChoiceAction generated for nimbus, we can identify the toolbar bottom and top selections and set the bottom selection as the default.